packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -18,30 +26,128 @@ export default class OperatorModeOverlays extends Component<Signature> {
+          />
+          {{#if @toggleSelect}}
+            <IconButton
+              {{on 'click' (fn @toggleSelect card)}}
+              class='hover-button select'
+              @icon={{if
+                (this.isSelected card.id)
+                'icon-circle-selected'
+                'icon-circle'
+              }}
+              aria-label='select card'
+            />
+          {{/if}}
+          <IconButton
+            class='hover-button preview'
+            @icon='eye'
+            aria-label='preview card'
+          />
+          <IconButton
+            class='hover-button more-actions'
+            @icon='more-actions'
+            aria-label='more actions'
+          />
+        </div>
+      {{/let}}
     {{/each}}
     <style>
-      .button { position: absolute; border: none; width: auto; }
+      .actions-overlay {
+        border-radius: var(--boxel-border-radius);
+      }
+      .actions-overlay:hover {
+        cursor: pointer;
+      }
+      .actions-overlay:hover {
+        box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.16);
+      }
+      .actions-overlay.selected {
+        box-shadow: 0 0 0 2px var(--boxel-highlight);
+      }
+      .overlay-button {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background: none;
+        border: none;
+        border-radius: inherit;
+        padding: 0;
+      }
+      .hover-button {
+        display: none;
+        position: absolute;
+        width: 30px;
+        height: 30px;
+      }
+      .actions-overlay:hover > .hover-button:not(:disabled),
+      .actions-overlay.selected > .hover-button.select {
+        display: block;
+      }
+      .hover-button:not(:disabled):hover {
+        --icon-color: var(--boxel-highlight);
+        --icon-border: var(--boxel-highlight);
+      }
+      .hover-button.select {
+        top: 0;
+        right: 0;
+      }
+      .hover-button.preview {
+        top: 0;
+        left: 0;
+      }
+      .hover-button.more-actions {
+        bottom: 0;
+        right: 0;
+      }
+      .hover-button > svg {
+        height: 100%;
+      }
     </style>
   </template>
 
   offset = {
     name: 'offset',
-    fn: (state: any) => {
-      let rects = state.rects;
+    fn: (state: MiddlewareState) => {
+      let padding = 20;
+      let { elements, rects } = state;
+      let { floating, reference } = elements;
+      let { width, height } = reference.getBoundingClientRect();
+      floating.style.width = width + 2 * padding + 'px';
+      floating.style.height = height + 2 * padding + 'px';
+
       return {
-        x: rects.reference.x + rects.reference.width - rects.floating.width,
-        y: rects.reference.y,
+        x: rects.reference.x - padding,
+        y: rects.reference.y - padding,
       };
     },
   };
+
+  @action openCard(card: Card) {
+    if (this.args.toggleSelect && this.args.selectedCards?.length) {
+      this.args.toggleSelect(card);
```

Comment:
There is some discrepancy between the name of this action and its outcome. So the name implies something will be opened but in this case it will be selected. Can this be improved by renaming the action, maybe to `selectCard`?